tests/relayer_integration_test.go

```diff
@@ -24,3 +29,22 @@ func (s *IntegrationTestSuite) TestERC20_Success_ClaimRelayer() {
 		relayerCommission,
 	)
 }
+
+func (s *IntegrationTestSuite) TestXMRMaker_DiscoverRelayer() {
+	ctx := context.Background()
+	c := rpcclient.NewClient(ctx, defaultXMRMakerSwapdEndpoint)
+	time.Sleep(time.Second * 30)
+
+	// see https://github.com/AthanorLabs/go-relayer/blob/master/net/host.go#L20
+	peerIDs, err := c.Discover("", defaultDiscoverTimeout)
+	require.NoError(s.T(), err)
+	s.T().Log(len(peerIDs))
```

Comment:
If the log is valuable, can you change it to `Logf` and add a context message?

Reply:
it's not, removed